Orientdb instruction

```diff
@@ -0,0 +1,20 @@
+Note this is just a proof of concept for using Orientdb an NoSQL graph database for this project.
+
+VirtualBox and Vagrant are required for this. The latest packages are found in the following URL:
+https://www.virtualbox.org/wiki/Downloads
+https://www.vagrantup.com/downloads.html
+
+After installing VirtualBox and Vagrant, using command prompt or Powershell, go to Vagrant location ("\phageParser\vagrant\") and just run "vagrant up".(If the Vagrant box does not download automatically, then run the command "vagrant box add ubuntu/trusty64") This will create a VM with Orientdb and RabbitMQ to the IP address 192.168.56.8.
+
+After the VM has been booted up, go into the "\phageParser\database" location. Run the following python files in order:
+create_database.py
```

Comment:
Hi @jialutu, first, thanks again for this! So far everything has installed and run okay. Thanks for the clear instructions.

At this step - 'python create_database.py' - I'm getting "TypeError: not all arguments converted during string formatting". Am I missing an argument or something?